### PR TITLE
HAWKULAR-47: Exit build on TypeScript errors

### PR DESCRIPTION
**Pull request by vrockai**

### CODE REVIEW

File: build/gulpfile.js [hunk 1]
```diff
@@ -56,12 +58,17 @@ module.exports = function(gulp, config, pluginName){
         module: 'commonjs',
         declarationFiles: true,
         noExternalResolve: false,
-        removeComments: true
-      })))
-      .on('error', plugins.notify.onError({
```
Comment (mtho11) [inline build/gulpfile.js:61]: Removing the notifier?

Comment (vrockai) [inline build/gulpfile.js:61]: this serves as a notifier: https://github.com/hawkular/hawkular-ui-components/pull/74/files#diff-b20c7230de4da834e815c06534cab810R63

Comment (mtho11) [inline build/gulpfile.js:61]: that just outputs to console though, right? no _toast_ notification messages pop up in the corner of your screen?

Comment (vrockai) [inline build/gulpfile.js:61]: ah, i don't have such notifications supported on my system. i can put that back, i just liked the output of the reported (in console) more.

Comment (mtho11) [inline build/gulpfile.js:61]: Thanks, I have learned to really like the notifications as I have the console running in some back window and have my IDE in fullscreen, so I just listen for a ding and the toasts fly across my screen. I don't have to look at console window ;)

**Pull request merged**